src/topology/connected.lean

```diff
@@ -770,7 +777,12 @@ lemma connected_component_rel_iff {x y : α} : ⟦x⟧ = ⟦y⟧ ↔
   connected_component x = connected_component y :=
 ⟨λ h, quotient.exact h, λ h, quotient.sound h⟩
 
-/-- The quotient of a space by its connected components -/
+lemma connected_component_nrel_iff {x y : α} : ⟦x⟧ ≠ ⟦y⟧ ↔
+  connected_component x ≠ connected_component y :=
+by {rw not_iff_not, exact connected_component_rel_iff}
+
+notation `π₀ ` α :max := quotient (connected_component_setoid α)
+/-- The quotient of a space by its connecte components -/
```

Comment:
```suggestion
by { rw not_iff_not, exact connected_component_rel_iff }

/-- The quotient of a space by its connected components -/
```
Looks like some deleted stuff got re-added in the merge.

Reply:
Forgot to mention: to fix the build error you'll have to `open_locale topological_space` somewhere in `src/topology/category/Profinite.lean`.

Reply:
I stopped using the notation anyway, since it was apparently non-standard. Now its called connected_components and everything compiles :)